src/app/researcher/study/request/[orgSlug]/actions.ts

```diff
@@ -17,12 +19,17 @@ const onCreateStudyActionArgsSchema = z.object({
 
 export const onCreateStudyAction = researcherAction(async ({ orgSlug, studyInfo }) => {
     const userId = await getUserIdFromActionContext()
+    const user = await currentUser()
+    if (!user) {
+        throw new Error('No user')
+    }
```

Comment:
`researcherAction` will require a user, it calls userAction internally. Given that you can call `actionContext()` to get the SiUser which should have name, etc.

Reply:
user is totally unused here, i just realized we can remove it entirely :)